packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -43,7 +44,7 @@ PatternFly has two implementations of a React table.
 
 The first is the newer `TableComposable` component. It takes a more explicit and declarative approach, and its implementation more closely mirrors that of an HTML table. Generally, updates and new feature requests are implemented in the `TableComposable`.
 
-The second is the original `Table` component. It is configuration based and takes a less declarative and more implicit approach about laying out the table structure, such as the rows and cells within it. Its documentation can be found <a href="/components/table/react-legacy">here</a>.
+The second is the original `Table` component. It is configuration based and takes a less declarative and more implicit approach about laying out the table structure, such as the rows and cells within it. Its documentation can be found <Link to="/components/table/react-legacy">here</Link>.
```

Comment:
Since the PR is already open, could be worth updating the last sentence to something along the lines of `Its documentation can be found on our <Link ...>legacy Table</Link> page.`